lametro/templates/lametro/person.html

```diff
@@ -157,7 +157,7 @@ <h3>
 
             {% if request.GET.view == 'committees' %}
                 <h3>Committees</h3>
-                {% if person.chair_role_memberships or person.member_role_memberships %}
+                {% if memberships_list %}
```

Comment:
So much nicer! Seems like there must be less overhead for coming up with this list than the previous way, too.

Reply:
yeah, perhaps! I just needed template code that was easier to digest.....